### ISSUE TITLE
Alphabetical sorting of lists with tasks complete and incomplete processes incorrectly

### ISSUE THREAD

**Issue by pcuellar** [Vinzent03/obsidian-sort-and-permute-lines#7]
I have a list of tasks with items complete and incomplete `[x] and [ ]`. This scenario resolves to complete tasks being sent to the bottom of the page instead of being properly organized alphabetically. Here is an example of list after being sorted alphabetically:

```
- [ ] Smith, Linda Tuhiwai...
- [ ] Tzul, Gladys Tzul...
- [x] Emiliani, Maria E. Posse...
- [x] Hawkes, Gina Louise, David Pollock, Barry Judd, Peter Phipps, and Elinor Assoulin...
- [ ] Younging, Gregory...
```

**Comment by Vinzent03** [comment]
Should be fixed in [0.2.2](https://github.com/Vinzent03/obsidian-sort-and-permute-lines/releases/tag/0.2.2)

**Comment by asitemade4u** [comment]
Thanks, both are legit ;o))
